ext/cfx-ui/src/app/game.service.ts

```diff
@@ -358,8 +360,12 @@ export class CfxGameService extends GameService {
 	private profileList: Record<string, string> = {};
 	card: boolean;
 
-	constructor(private sanitizer: DomSanitizer, private zone: NgZone, private translation: L10nTranslationService) {
+	private serversService: ServersService;
+
+	constructor(private sanitizer: DomSanitizer, private zone: NgZone, private translation: L10nTranslationService,
+				private httpClient: HttpClient, @Inject(PLATFORM_ID) private platformId: any) {
 		super();
+		this.serversService = new ServersService(httpClient, sanitizer, zone, this, platformId);
```

Comment:
This seems fishy. Even if ServersService depends on GameService, might it make more sense instead to factor out the dependencies needed by both into a separate service (e.g. ServersCacheService), potentially forwarding calls from ServersService into this new service to be compatible with downstream consumers, instead of being weirdly recursive or doing this _completely weird hack_?